extensions/binding/org.eclipse.smarthome.binding.openweathermap/README.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# OpenWeatherMap Binding
+
+This binding integrates the OpenWeatherMap Weather API (see https://openweathermap.org/api).
+
+## Supported Things
+
+There are four supported things.
+
+### OpenWeatherMap API
+
+First one is a bridge `weather-api` which represents the connection to the API.
+The bridge holds the mandatory API key to access the OpenWeatherMap API and several global configuration parameters.
+
+### Current Weather
+
+The second thing `weather` supports the current weather service for a specific location (see https://openweathermap.org/current).
+It requires coordinates of the location of your interest.
+You can add as much `weather` things for different locations to your setup as you like to observe.
+
+### 3 Hour Forecast
+
+Third thing is `hourly-forecast` which supports 5 day / 3 hour forecast (currently for 24h, see https://openweathermap.org/forecast5).
+It requires coordinates of the location of your interest.
+You can add as much `hourly-forecast` things for different locations to your setup as you like to observe.
+
+### Daily Forecast
+
+The `daily-forecast` thing retrieves 16 day / daily forecast data (currently for 5 days, see https://openweathermap.org/forecast16).
+It requires coordinates of the location of your interest.
+You can add as much `daily-forecast` things for different locations to your setup as you like to observe.
+**Attention**: The daily forecast is only available for all paid accounts (see https://openweathermap.org/price).
+
+## Discovery
+
+At first the auto-discovery will add an "OpenWeatherMap API" `weather-api` bridge.
```

Comment:
That is something I do not understand. Why? We did the same in #5501.

Reply:
> That is something I do not understand. Why?

Because discovery should only discover what really exists. Imagine you and your spouse each have an OWM account. Which one do you discover? Imagine myself having no account? Which one did you discover? Discovery must not be misused as a shortcut to prompt users to manually create a thing of a certain type.

> We did the same in #5501.

Did we? I can only see [that it discovers the weather for the system location](https://github.com/eclipse/smarthome/pull/5501/files#diff-16d0660129ea02eeab1ec854d8eeb244R23), once a bridge (=account) has been (manually) created.